opm/simulators/flow/SimulatorFullyImplicitBlackoil.hpp

```diff
@@ -420,14 +426,7 @@ class SimulatorFullyImplicitBlackoil : private SerializableSim
             // Grab the step convergence reports that are new since last we
             // were here.
             const auto& reps = this->solver_->model().stepReports();
-
-            auto reports = std::vector<StepReport> {
-                reps.begin() + this->already_reported_steps_, reps.end()
-            };
-
-            this->writeConvergenceOutput(std::move(reports));
-
-            this->already_reported_steps_ = reps.size();
+            convergence_output_.write(std::move(reps));
```

Comment:
Tiny nit: This should be `write(reps)` instead of `write(move(reps))`.  Given that `reps` is a reference-to-`const` and that `SimulatorConvergenceOutput::write()` takes a reference-to-`const`, the extra `move()` here is misleading.

Reply:
yeah. leftover, as i changed to do the range extract on the inside..